modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/A11yTool.testcase

```diff
@@ -93,6 +98,75 @@ definition {
 		takeScreenshot();
 	}
 
+	@description = "LPS-141820. Verifies violations can be filtered by 'Critical' only."
+	@priority = "5"
+	test CanFilterViolationsByCriticalImpactOnly {
+		property osgi.module.configuration.file.names = "com.liferay.frontend.js.a11y.web.internal.configuration.FFA11yConfiguration.config";
+		property osgi.module.configurations = "enable=B&quot;true&quot;";
+
+		// Workaround for side panel sometimes not displaying LPS-141442
+
+		while (IsElementNotPresent(locator1 = "A11yTool#VIOLATION_PANEL_HEADER")) {
+			Refresh();
+		}
+
+		task ("Verify page has Minor, Moderate, Serious, and Critical set of violations") {
+			for (var severity : list "minor,moderate,serious,critical") {
+				AssertElementPresent(
+					key_text = "${severity}",
+					locator1 = "A11yTool#VIOLATION_PANEL_ITEM_SEVERITY");
+			}
```

Comment:
Remove the for loop. And change assert to verify. I think verify is better for setup phases.

The assertion for this should be covered by A11yTool#CanFilterViolationsByAllImpact